vllm/config.py

```diff
@@ -94,6 +94,11 @@ def get_head_size(self) -> int:
         return self.hf_config.hidden_size // self.hf_config.num_attention_heads
 
     def get_num_heads(self, parallel_config: "ParallelConfig") -> int:
+        if getattr(self.hf_config, "multi_query", False):
+            # Multi-query attention, only one KV head.
+            return 1
+        if getattr(self.hf_config, "n_head_kv", None) is not None:
+            return self.hf_config.n_head_kv
```

Comment:
A quick question: Is this only used in `GPTBigCode`? or Is it generally used for other MQA models?

Reply:
`multi_query` is used by  GPTBigCode and  `n_head_kv` is used by Falcon. I think for now this is OK and we can add more `if`s here for newer models.

Reply:
Got it. Thanks!